resources/server_install.rb

```diff
@@ -0,0 +1,112 @@
+# To learn more about Custom Resources, see https://docs.chef.io/custom_resources.html
+#
+# Author:: Corey Hemminger
```

Comment:
I do believe some credit should be given to me here, as a lot of this code looks copy/pasted from recipes that I developed in the original repo.

Reply:
agreed, that was copy pasted from another cookbook I had to start my resource file.

Reply:
reverted my changes and moved my changes to a new fork TheREK3R#1